es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,58 @@
+
+
+import TodoList from './todo-list';
+import TodoForm from './todo-form';
+import createElement from '../lib/create-element';
+import noop from '../lib/noop';
```

Comment:
Here you want to `import` these from the `lib/index.js` file:
```JAVASCRIPT
import { createElement, noop } from '../lib';
```

Reply:
updated